crypto/keys/keybase_base.go

```diff
@@ -185,37 +211,19 @@ func (kb baseKeybase) CreateMnemonic(
 		return nil, "", err
 	}
 
-	info, err = kb.persistDerivedKey(
-		keyWriter,
-		bip39.NewSeed(mnemonic, DefaultBIP39Passphrase), passwd,
-		name, types.GetConfig().GetFullFundraiserPath(),
-	)
+	info, err = kb.CreateAccount(keyWriter, name, mnemonic, DefaultBIP39Passphrase, passwd, types.GetConfig().GetFullFundraiserPath(), algo)
```

Comment:
check for error

Reply:
If it returns right away, why do I need to check for error?

Reply:
yes because then you return a `nil` info and empty `""` mnemonic